samples/lightweight-component/lightweight_component.ipynb

```diff
@@ -0,0 +1,359 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Lightweight python components example\n",
+    "\n",
+    "This pipeline defines two simple math operation: `add` adds two input arguments and returns the sum. `my_divmod` divides two input arguments and returns the quotient and remainder. You can visit [Build Lightweight Python Components](https://www.kubeflow.org/docs/pipelines/sdk/lightweight-python-components/) for this pipeline's detail."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "pipeline_name = \"lightweight-python-demo\"\n",
+    "user_namespace = \"myproject\""
```

Comment:
I was trying it out on a cluster without Kubeflow and it failed. Maybe use default namespace until we made Kubeflow as part of the requirements?